lib/pages/settings.dart

```diff
@@ -226,6 +226,18 @@ class _SettingsPageState extends State<SettingsPage> {
                             height16,
                             colourDropdown,
                             height16,
+                            Row(
+                              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                              children: [
+                                Text(tr('amoledTheme')),
+                                Switch(
+                                    value: settingsProvider.amoledDarkTheme,
+                                    onChanged: (value) {
+                                      settingsProvider.amoledDarkTheme = value;
+                                    })
+                              ],
+                            ),
+                            height16,
```

Comment:
`height16` won't be necessary anymore as the `colourDropdown` has enough space above it.